### PR TITLE
Made the first line bold

### PR DESCRIPTION
**Pull request by rishav142k**
first line is bold and removed the *first* remark as there is only one point there.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Contributing Guide
 
-*First*: if you feel insecure about how to start contributing, feel free to ask us on our [Discord channel](https://discordapp.com/invite/hasura), in the #contrib channel. You can also just go ahead with your contribution and we'll give you feedback. Don't worry - the worst that can happen is that you'll be politely asked to change something. We appreciate any contributions, and we don't want a wall of rules to stand in the way of that. 
+**: If you feel insecure about how to start contributing, feel free to ask us on our [Discord channel](https://discordapp.com/invite/hasura), in the #contrib channel**. You can also just go ahead with your contribution and we'll give you feedback. Don't worry - the worst that can happen is that you'll be politely asked to change something. We appreciate any contributions, and we don't want a wall of rules to stand in the way of that.
```
Comment (praveenweb) [inline CONTRIBUTING.md]: Let's remove only the first and keep the rest of the text in regular font weight.

Comment (rishav142k) [reply CONTRIBUTING.md]: Okay, so how do I do it? I mean, I am new to open source and github, in general and I don't know how to edit a commit. Do I open a new PR now?

Comment (praveenweb) [reply CONTRIBUTING.md]: Hope you figured out that you can keep pushing to the same PR :)

**Comment by marionschleifer** [comment]
@rishav142k could you please send an email to marion@hasura.io, so that we can arrange the shipping of your swag? 🙂

**Pull request merged**